modules/backup-and-restore/pages/backup-cluster.adoc

```diff
@@ -39,9 +39,15 @@ $ gadmin backup create weekly
 
 This creates a backup file named `weekly-<timestamp>` at the xref:configurations.adoc[backup location you configured].
 
+
 [#_metadata_backup_optional]
 === Metadata backup (Optional)
 
+IMPORTANT: Since v3.9.3, it is no longer required to create a backup for a backups metadata.
```

Comment:
since v3.9.2.

Reply:
Thank you. Changed this with latest commit.